modules/lvms-about-adding-devices-to-a-vg.adoc

```diff
@@ -6,22 +6,32 @@
 [id="about-adding-devices-to-a-vg_{context}"]
 = About adding devices to a volume group
 
-The `deviceSelector` field in the `LVMCluster` CR contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
+The `deviceSelector` field in the `LVMCluster` CR contains the configuration to specify the paths to the devices that you want to add to the LVM volume group (VG).
 
 You can specify the device paths in the `deviceSelector.paths` field, the `deviceSelector.optionalPaths` field, or both. If you do not specify the device paths in both the `deviceSelector.paths` field and the `deviceSelector.optionalPaths` field, {lvms} adds the supported unused devices to the LVM volume group. 
 
-The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see "Devices not supported by {lvms}" in the "Additional resources" section.
+You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see "Integrating RAID arrays with {lvms}".
 
-If you do not add the `deviceSelector` field in the `LVMCluster` CR, {lvms} automatically adds the new devices when the devices are available.
+The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see "Devices not supported by {lvms}".
 
 {lvms} adds the devices to the LVM volume group only if the following conditions are met:
 
 * The device path exists.
-* The device is supported by {lvms}.
-
-You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see "Integrating RAID arrays with {lvms}" in the "Additional resources" section. 
+* The device is supported by {lvms}. 
 
 [IMPORTANT]
 ====
-After a device is added to the LVM volume group, it cannot be removed.
-==== 
\ No newline at end of file
+After a device is added to the VG, it cannot be removed.
+==== 
+
+{lvms} supports dynamic device discovery. If you do not add the `deviceSelector` field in the `LVMCluster` CR, {lvms} automatically adds the new devices when the devices are available.
+
+[WARNING]
+====
+It is not recommended to use dynamic device discovery due to the following reasons:
+
+* When you add a new device that you do not intend to add to the VG, {lvms} automatically adds this device to the VG through dynamic device discovery if the `deviceSelector` field is not configured.
+* After a device is added to the VG by using the `deviceSelector` field, you cannot remove the device. However, if {lvms} adds a device to the VG through dynamic device discovery, {lvms} does not restrict you from removing the device from the node. Removing or updating the devices that are already added to the VG can disrupt the VG. This can also lead to data loss and necessitate manual node remediation.
```

Comment:
"Dynamic device discovery" is similar to "Dynamic provisioning". So I think just "through dynamic device discovery" would be more clear?

Data loss happens after the VG is disrupted. So I guess considering the order of events, it will be best to have a separate line to talk about data loss and manual node remediation.

WDYT?

Reply:
Sounds good